users/sam-id-b3/wpm_oled.c

```diff
@@ -0,0 +1,86 @@
+#include "wpm_oled.h"
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
Thank you for the review. I've implemented all suggested changes and have updated my fork and branch to match with master.